### ISSUE TITLE
Configure SystemJS to use flex-layout.umd.js

### ISSUE THREAD

**Issue by esff** [angular/flex-layout#42]
Started the demo application and it worked fine. Then, from the dist folder,  made a copy of @angular/flex-layout to my running application setup generated from the "Angular Docs 5 Min Quickstart". 

Registered FlexLayoutModule on app.module, as described on documentation and  added '@angular/flex-layout': 'npm:@angular/flex-layout/flex-layout.umd.js' to mappings on Systemjs.config.js

Unfortunately it returned an "ReferenceError: Invalid left-hand side in assignment" on Chrome, Firefox, IE and Opera BUT worked great on Edge. Any thoughts on how to fix this?

**Comment by germolinal** [comment]
Same here! 

`gulp build:components` seems to work well. Then I copied the flex-layout folder from the `dist/@angular` to the `node-modules` folder in my project, and I get this error.

```
Error: (SystemJS) invalid assignment left-hand side
	ZoneDelegate.prototype.invoke@http://localhost:8000/node_modules/zone.js/dist/zone.js:232:17
	Zone.prototype.run@http://localhost:8000/node_modules/zone.js/dist/zone.js:114:24
	scheduleResolveOrReject/<@http://localhost:8000/node_modules/zone.js/dist/zone.js:502:52
	ZoneDelegate.prototype.invokeTask@http://localhost:8000/node_modules/zone.js/dist/zone.js:265:21
	Zone.prototype.runTask@http://localhost:8000/node_modules/zone.js/dist/zone.js:154:28
	drainMicroTaskQueue@http://localhost:8000/node_modules/zone.js/dist/zone.js:401:25
	ZoneTask/this.invoke@http://localhost:8000/node_modules/zone.js/dist/zone.js:339:25
	
	Evaluating http://localhost:8000/node_modules/@angular/flex-layout/flex-layout.umd.js
	Error loading http://localhost:8000/node_modules/@angular/flex-layout/flex-layout.umd.js as "@angular/flex-layout" from http://localhost:8000/app/app.module.js
Apilar seguimientos:
(SystemJS) invalid assignment left-hand side
	ZoneDelegate.prototype.invoke@http://localhost:8000/node_modules/zone.js/dist/zone.js:232:17
	Zone.prototype.run@http://localhost:8000/node_modules/zone.js/dist/zone.js:114:24
	scheduleResolveOrReject/<@http://localhost:8000/node_modules/zone.js/dist/zone.js:502:52
	ZoneDelegate.prototype.invokeTask@http://localhost:8000/node_modules/zone.js/dist/zone.js:265:21
	Zone.prototype.runTask@http://localhost:8000/node_modules/zone.js/dist/zone.js:154:28
	drainMicroTaskQueue@http://localhost:8000/node_modules/zone.js/dist/zone.js:401:25
	ZoneTask/this.invoke@http://localhost:8000/node_modules/zone.js/dist/zone.js:339:25
	
	Evaluating http://localhost:8000/node_modules/@angular/flex-layout/flex-layout.umd.js
	Error loading http://localhost:8000/node_modules/@angular/flex-layout/flex-layout.umd.js as "@angular/flex-layout" from http://localhost:8000/app/app.module.js
```

**Comment by germolinal** [comment]
I also saw this when compiling with NTSC in Windows 10

```
client/node_modules/@angular/flex-layout/media-query/media-monitor.d.ts(4,36): error TS2306: File 'C:/Users/German/Documents/GH_Backend/client/node_modules/@angular/flex-layout/media-query/breakpoints/break-point-registry.d.ts' is not a module.
client/node_modules/@angular/flex-layout/utils/index.d.ts(1,15): error TS2306: File 'C:/Users/German/Documents/GH_Backend/client/node_modules/@angular/flex-layout/utils/auto-prefixer.d.ts' is not a module.
client/node_modules/@angular/flex-layout/utils/index.d.ts(2,15): error TS2306: File 'C:/Users/German/Documents/GH_Backend/client/node_modules/@angular/flex-layout/utils/object-extend.d.ts' is not a module.
```

**Comment by ThomasBurleson** [comment]
Fixed with SHA 2a1590a30063a58734951f67f3f6c4bbe71090d7 + SHA 3f2aa2167efc741f8eab27d8198b17f54a22c21d.

> Note:  `gulp build:lib` and `gulp build:release` are now used.

After `gulp build:release`, if you copy the dir from `/dist/@angular/flex-layout` and paste into your `node_modules/@angular/` directory, you should be able to use the Flex Layout API with the Angular CLI.

Please retry with the lastest from **#master**.
